Application/EdFi.Ods.Security/Claims/ClaimsIdentityProvider.cs

```diff
@@ -48,47 +47,39 @@ public ClaimsIdentity GetClaimsIdentity()
             IEnumerable<string> namespacePrefixes,
             IReadOnlyList<string> assignedProfileNames)
         {
-            var filteredNamespacePrefixes = namespacePrefixes.Where(np => !string.IsNullOrWhiteSpace(np))
-                                                             .ToList();
+            var nonEmptyNamespacePrefixes = namespacePrefixes.Where(np => !string.IsNullOrWhiteSpace(np)).ToList();
 
             var resourceClaims = _securityRepository.GetClaimsForClaimSet(claimSetName);
 
             // Group the resource claims by name to combine actions (and by claim set name if multiple claim sets are supported in the future)
-            var resourceClaimsByClaimName =
-                from c in resourceClaims
-                group c by c.ResourceClaim.ClaimName
-                into g
+            var resourceClaimsByClaimName = from c in resourceClaims
```

Comment:
this should be converted to the linq select code

Reply:
Done.